.circleci/config.yml

```diff
@@ -87,24 +123,42 @@ jobs:
 
     steps:
       - run:
-          name: Install dependencies
+          name: install dependencies
           command: |
             yum install -y git centos-release-scl cmake libsndfile make bzip2 wget which unzip
             yum-config-manager --enable rhel-server-rhscl-7-rpms
             yum install -y devtoolset-7-gcc-c++
       - checkout
       - run:
-          name: run tests
+          name: install espnet
           command: |
             # NOTE(karita) this line exited 1
             # source scl_source enable devtoolset-7
             PATH="/opt/rh/devtoolset-7/root/usr/bin:${PATH:-}"
             ./ci/install.sh
+      - run:
+          name: test shell
+          command: |
+            PATH="/opt/rh/devtoolset-7/root/usr/bin:${PATH:-}"
             ./ci/test_shell.sh
+      - run:
+          name: test python
+          command: |
+            PATH="/opt/rh/devtoolset-7/root/usr/bin:${PATH:-}"
             ./ci/test_python.sh
-            # TODO(karita) centos7 featbin binary
-            # ./ci/install_kaldi.sh
-            # ./ci/test_integration.sh
+      # - run:
+      #     name: install kaldi
+      #     command: |
+      #       # TODO(karita) centos7 featbin binary
+      #       PATH="/opt/rh/devtoolset-7/root/usr/bin:${PATH:-}"
+      #       ./ci/install_kaldi.sh
+      - run:
+          name: test integration
+          command: |
+            ./ci/test_integration.sh
```

Comment:
You cannot run this in CentOS7 because I haven't prepared kaldi featbin for CentOS7 (old libc distribution).